kube/services/revproxy/nginx.conf

```diff
@@ -424,6 +424,13 @@ server {
       include /etc/nginx/gen3.conf/gen3ff-as-root/*.conf;
   }
 
+    location @403error {
+      if ($frontend_root_service = "gen3ff") {
+        return 302 https://$host/portal/403error;
+      }
+      return 302 https://$host/403error;
+  }
```

Comment:
@nss10 instead of calling portal here, perhaps display a more generic message?

Reply:
I can surely try that, I was hoping if someone can point me to a use case where a non-workspace related 403 error occurs.